src/components/mapView/index.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { MapContainer, Marker, Popup, TileLayer } from 'react-leaflet';
+import { Icon } from 'leaflet';
+import 'leaflet/dist/leaflet.css';
+
+import locationIcon from 'assets/locationIcon.svg';
+
+import './styles.scss';
+
+const userLocationIcon = new Icon({
+  iconUrl: locationIcon,
+  iconRetinaUrl: locationIcon,
+  popupAnchor: [-0, -0],
+  iconSize: [53, 69],
+});
+
+const MapView = ({ location }) => {
+  const { lat, lon } = location?.coordinates;
+
+  return (
+    <>
+      {location.loaded && !location.error && (
```

Comment:
I think that it would be a better approach to always show the map, even if the location isn't loaded or it has errors. Maybe add this validation to render just the marker. 

Maybe you can add a default location to center the map in case the geolocation fails.